Classes/ServiceProvider.php

```diff
@@ -187,6 +187,13 @@ public static function getContentBlockTypoScript(ContainerInterface $container):
                     if (!file_exists($absolutePath)) {
                         continue;
                     }
+
+                    $importStatement = '';
+                    if (file_exists(GeneralUtility::getFileAbsFileName($extPath . '/setup.typoscript'))) {
+                        $importStatement = '@import "' . $extPath . '/setup.typoscript"';
```

Comment:
One might want to adjust the path, e.g.:

```suggestion
                    if (file_exists(GeneralUtility::getFileAbsFileName($extPath . '/typoscript/setup.typoscript'))) {
                        $importStatement = '@import "' . $extPath . '/typoscript/setup.typoscript"';
```